PROMPT-MR13-httpclient-retry-tail.md

```
@@ -0,0 +1,177 @@
+
+| Phase              | phaseMs    | cumulativeMs | Note                                         |
+| ------------------ | ---------- | ------------ | -------------------------------------------- |
+| fetchRep           | 620        | 620          |                                              |
+| fetchContributions | 4,478      | 5,099        | FEC was fast this time                       |
+| **fetchVotes**     | **53,330** | **53,951**   | **2 /members fetches hit retry tail**        |
+| fetchUpstream      | 53,331     | 53,951       | (max of votes + contributions)               |
+| ...                |            |              | analyzer timed out at 55s, nothing past here |
+
+Vercel logs from the same run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)